gtwrap/interface_parser/classes.py

```diff
@@ -12,7 +12,7 @@
 
 from typing import Iterable, List, Union
 
-from pyparsing import Literal, Optional, ZeroOrMore
+from pyparsing import Literal, Optional, ZeroOrMore  # type: ignore
```

Comment:
Note to self: generate typing annotations for pyparsing

Reply:
How can we generate annotations for external libraries?

Reply:
`stubgen`